tests/core18/compat/task.yaml

```diff
@@ -1,11 +1,11 @@
 summary: Ensure that core(16) compatibility is there
 
 execute: |
-    echo "Install test-snapd-tools (which uses the core snap)"
-    snap install test-snapd-tools
+    echo "Install test-snapd-sh (which uses the core snap)"
+    snap install test-snapd-sh
 
     echo "Ensure that this pulled in core"
     snap list | MATCH "^core +"
 
-    echo "Check test-snapd-tools see the core16 environment"
-    test-snapd-tools.cat /etc/os-release | MATCH "Ubuntu Core 16"
+    echo "Check test-snapd-sh see the core16 environment"
+    test-snapd-sh -c 'cat /etc/os-release' | MATCH "Ubuntu Core 16"
```

Comment:
This should be `tests-snapd-sh.sh ..`. I'll push a patch to fix it.

Reply:
@bboozzoo nice, thanks